developers/weaviate/client-libraries/python/async.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Async API
+sidebar_position: 40
+image: og/docs/client-libraries.jpg
+# tags: ['python', 'client library']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
+import FastAPIExample from '!!raw-loader!/_includes/code/client-libraries/minimal_fastapi.py';
+
+:::info Added in `weaviate-client` `v4.7.0`
+The async Python client is available in `weaviate-client` versions `4.7.0` and higher.
+:::
+
+The Python client library provides a [synchronous API](./index.md) by default, but an asynchronous API is also available for concurrent applications.
+
+For asynchronous operations, use the `WeaviateAsyncClient` async client, available in `weaviate-client` `v4.7.0` and up.
+
+The `WeaviateAsyncClient` async client largely supports the same functions and methods as the `WeaviateClient` [synchronous client](./index.md), with the key difference that the async client is designed to be used in an `async` function or in an [`asyncio` event loop](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio-event-loop).
```

Comment:
> used in an `async` function or in an `asyncio event loop`

I would change this to
> used in an `async` funcion running in an `asyncio event loop`

This is because calling `await` must always be within an `async` function and an `async` function only executes correctly if it is called within a running event loop, so it's not mutually exclusive